src/main/java/GuiFileChooser.java

```diff
@@ -21,7 +22,8 @@ public File getDirectoryFromUser() {
         JFrame jFrame = new JFrame("FileChooserDemo");
         jFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         JFileChooser fc = new JFileChooser();
-        fc.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+        fc.setFileFilter(new FileNameExtensionFilter("SB2 Files", "sb2"));
```

Comment:
I like this way this filters to only show and allow selection of sb2s.  However, when I run this, it looks like the file chooser is asking me to select SB2 Files, when what we really is a directory containing sb2s.  What would y'all think of changing "SB2 Files" to "Folder Containing SB2 Files"?